apps/backend/src/controllers/agri/bpp/confirm.ts

```diff
@@ -0,0 +1,214 @@
+import { NextFunction, Request, Response } from "express";
+import {
+	checkIfCustomized,
+	redisFetchFromServer,
+	responseBuilder,
+	send_nack,
+	Stop,
+	updateFulfillments,
+} from "../../../lib/utils";
+import { ON_ACTION_KEY } from "../../../lib/utils/actionOnActionKeys";
+import { ORDER_STATUS } from "../../../lib/utils/apiConstants";
+import { ERROR_MESSAGES } from "../../../lib/utils/responseMessages";
+
+export const confirmController = (
+	req: Request,
+	res: Response,
+	next: NextFunction
+) => {
+	try {
+		if (checkIfCustomized(req.body.message.order?.items)) {
+			return confirmServiceCustomizationController(req, res, next);
+		}
+		
+		confirmConsultationController(req, res, next);
+	} catch (error) {
+		return next(error);
+	}
+};
+
+export const confirmConsultationController = async (
+	req: Request,
+	res: Response,
+	next: NextFunction
+) => {
+	try {
+		const {
+			context,
+			message: { order },
+		} = req.body;
+
+		const on_init = await redisFetchFromServer(
+			ON_ACTION_KEY.ON_INIT,
+			context?.transaction_id
+		);
+
+		if (on_init && on_init?.error) {
+			return send_nack(
+				res,
+				on_init?.error?.message
+					? on_init?.error?.message
+					: ERROR_MESSAGES.ON_INIT_DOES_NOT_EXISTED
+			);
+		}
+
+		if (!on_init) {
+			return send_nack(res, ERROR_MESSAGES.ON_INIT_DOES_NOT_EXISTED);
+		}
+
+		const { fulfillments } = order;
+		const responseMessage = {
+			order: {
+				...order,
+				status: ORDER_STATUS.ACCEPTED.toUpperCase(),
+				fulfillments: [{
+					...fulfillments[0],
+					start: {
+						contact: {
+							email: "tech-support@test.com",
```

Comment:
Please remove hardcoding in favour of picking from example across all instances. Also keep in mind that time should not be picked from example. Time should be current times.